R/utils.R

```diff
@@ -459,7 +459,7 @@ srv_arbitrary_lines <- function(id) {
             )
           )
         }
-        if (length(line_arb_color) == 0 || line_arb_color == "") {
+        if (length(line_arb_color) == 0 || all(line_arb_color %in% "")) {
```

Comment:
```suggestion
        if (length(line_arb_color) == 0 || all(line_arb_color == "")) {
```

Does this not work?
![image](https://user-images.githubusercontent.com/15201933/178521290-b70b190c-63cc-4119-8736-6a0db52fdf13.png)